lambdamud-client/src/components/MainScreen.js

```diff
@@ -119,7 +119,7 @@ class MainScreen extends React.Component {
   render() {
     const playersArray = this.state.players.slice();
 
-    if (this.state.userInput === "n" || this.state.userInput == "e" || this.state.userInput == "w" || this.state.userInput == "s" ) {
+    if (this.state.userInput === "n" || this.state.userInput === "e" || this.state.userInput === "w" || this.state.userInput === "s" ) {
```

Comment:
Yeah, it's generally best to have === whenever possible although == usually is ok when comparing strings. I wonder why your console was printing errors.